packages/plugin-core/src/_extension.ts

```diff
@@ -976,21 +975,29 @@ export async function showLapsedUserMessage(assetUri: vscode.Uri) {
     });
 }
 
-export async function shouldDisplayInactiveUserSurvey(): Promise<boolean> {
-  const inactiveSurveySubmitted = await StateService.instance().getGlobalState(
-    GLOBAL_STATE.INACTIVE_USER_SURVEY_SUBMITTED
-  );
+export function shouldDisplayInactiveUserSurvey(): boolean {
+  const metaData = MetadataService.instance().getMeta();
 
-  // don't display if they have submitted before.
-  if (inactiveSurveySubmitted === "submitted") {
+  const inactiveSurveyMsgStatus = metaData.inactiveUserMsgStatus;
+  if (inactiveSurveyMsgStatus === "submitted") {
     return false;
   }
 
+  // rare case where global state has been reset (or a reinstall) may cause issues with
+  // the prompt logic. ignore these cases and don't show the
+  if (
```

Comment:
can you create a followup task to fix this in the future? syouldn't be part of this pr but we do want to fix these cases

Reply:
[[Metadata Service Is Unaware of Reinstalls|dendron://private/task.2022.03.14.metadata-service-is-unaware-of-reinstalls]]